src/components/content/medications-table-base.tsx

```diff
@@ -60,7 +60,6 @@ export const MedicationsTableBase = ({
   return (
     <div className={`ctw-table ${className}`.trim()}>
```

Comment:
Little weird that this also has `ctw-table`, I'd expect that to be specific to the table core component.

Reply:
yeah good point it should just be
```suggestion
  return (
    <div className={className}>
```